Bugzilla/Elastic/Indexer.pm

```diff
@@ -109,7 +104,7 @@ sub _bulk_helper {
     );
 }
 
-sub find_largest_mtime {
```

Comment:
Was this a cosmetic change uncovered while fixing the core problem, but not necessarily related to the problem itself?